### ISSUE TITLE
Users need to know they won't necessarily receive their pass within 5 days 

### ISSUE THREAD

**Issue by petegale** [alphagov/verify-local-patterns#189]
**Impact:** Having not received a pass, users would be more likely to contact the council, leading to increased workload and dissatisfied users.

**Evidence:** Several users stated thought that this page said they would receive their pass in 5 days, when in fact it said the pass would be dispatched in 5 days.

When scan-reading this page, the second line reads “bus pass within 5 days”

**Comment by petegale** [comment]
![screen shot 2017-03-03 at 15 19 10](https://cloud.githubusercontent.com/assets/2970019/23556497/c4ec2e04-0024-11e7-869f-dd0107c5274f.png)

**Comment by suryanaga** [comment]
We've reduced the impact of this by adding in notifications, which every user we've tested with has signed up for.

**Comment by lizziebruce** [comment]
@irinapencheva is this still happening after the notifications mentioned by Sanjay ^^ ?

Copy solution options:

> we'll usually send your bus pass within 5 working days

or

> we'll aim to send your bus pass within 5 working days

or

> you should receive your bus pass within 10 working days

**Comment by suryanaga** [comment]
@lizziebruce Yep, those options could work. @petegale's going to probe the impact of this issue next week.

**Comment by irinapencheva** [comment]
@lizziebruce - as discussed, I like the 2nd option and change it to 2 WDs

@sanjaypoyzer - this came up in last round of UR so we need this minor change

**Comment by suryanaga** [comment]
@irinapencheva Recent changes have been made that affect this need since the last round of UR.

**Comment by lizziebruce** [comment]
@sanjaypoyzer I've discussed this with @irinapencheva and the phrasing on the Checking your identity page time to arrive bullet needs to change to this for Verify:

> we'll aim to send your bus pass within 2 working days

and this for non-Verify:

> we'll aim to send your bus pass within 5 working days

This is because the council can only say what they will do when i.e. send the pass not when it will arrive as that depends on 3rd party – postal service.

Could you update this as I am having problems viewing local host and the Verify Choice page has proven to be complex for me to edit the code on?

**Comment by lizziebruce** [comment]
@sanjaypoyzer @irinapencheva following our discussion of Verify receive in 5 working days, non-Verify receive in working 10 days it looks like the facts on this page are actually incorrect. Shouldn't it be:

Before
> If your photo passes our manual checks, we'll send you your bus pass within 5 days. If it doesn't pass we'll contact you.

After
> If your photo passes our manual checks, you should usually receive your bus pass within 5 working days. If it doesn't pass we'll contact you.


I'd actually recommend changing the content of the top section to:

> ## What happens next
> 
> We’ll check that your photo has a plain white background, shows your shoulders and shows your face clearly. If it doesn't we'll contact you. 
>
> If your photo passes our check, you'll usually receive your free bus pass within 5 working days. 
> 
> We'll send it to this address:
> 
> Jane Smith
> 1A Argleton High Street
> Argleton
> ARG1 4HP
> Provide a different address
> 
> You’ll receive a confirmation email and text with this reference number.

**Comment by irinapencheva** [comment]
@lizziebruce - I like the 'What happens next' section. The only minor question I have is do we need to list the photo criteria again or could we say as above 'our manual checks', also given that many councils would pass the photo even without these criteria exactly matched? Either ways, I think it works well.

**Comment by lizziebruce** [comment]
@irinapencheva I've left off repeating the photo criteria, but this is something we could keep in mind for future suggestion: if councils find that in practice people question what manual checks means or get disgruntled when their photo fails. It hasn't come up in testing so agree, no requirement to include for now. 

I'd like to have a think about an alternative verb for "passes" checks, as with it being bus pass there's a lot of different usage of the word pass in the same of couple sentences..

Here's what that page looks like now. 

<img width="538" alt="screen shot 2017-05-24 at 10 26 35" src="https://cloud.githubusercontent.com/assets/27814324/26396942/981bfdea-406c-11e7-9ca8-6de43486618f.png">

**Comment by irinapencheva** [comment]
Agree on both points! Maybe for the 'pass manual checks' something around 'approvals'?
